Project/Assets/Editor/Twine Importer/TwineJsonParser.cs

```diff
@@ -1,12 +1,17 @@
 ﻿using UnityEngine;
 using System.Collections;
+using System.Collections.Generic;
 using SimpleJSON;
 
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
 public class TwineJsonParser {
 
 	// Keys for JSON:
-	private readonly static string NodeIdKey = "pid";
-	private readonly static string NodeNameKey = "name";
+	//private readonly static string NodeIdKey = "pid";
```

Comment:
Are we planning on ever using these fields? If not, I think let's get rid of it and try to avoid commented out code.